reviews/sandbox/2022/2022-Tinkerbell-annual.md

```diff
@@ -0,0 +1,81 @@
+
+| Maintainer        | GitHub Username                                      | Company       |
+| ----------------- | ---------------------------------------------------- | ------------- |
+| Manuel Mendez     | [@mmlb](https://github.com/mmlb)                     | Equinix Metal |
+| Micah Hausler     | [@micahhausler](https://github.com/micahhausler)     | AWS           |
+| Jacob Weinstock   | [@jacobweinstock](https://github.com/jacobweinstock) | AWS           |
+| Nahum Shalman     | [@nshalman](https://github.com/nshalman)             | Equinix Metal |
+| Chris Doherty     | [@chrisdoherty4](https://github.com/chrisdoherty4)   | AWS           |
+| Thomas Stromberg  | [@tstromberg](https://github.com/tstromberg)         | Chainguard    |
+| Daniel Finneran   | [@thebsdbox](https://github.com/thebsdbox)           | Isovalent     |
+| Jason DeTiberus   | [@detiber](https://github.com/detiber)               | Cisco         |
+| Marques Johansson | [@displague](https://github.com/displague)           | Equinix Metal |
+| Jeremy Tanner     | [@jeremytanner](https://github.com/jeremytanner)     | Tailscale     |
+| Mara              | [@DailyAlice](https://github.com/DailyAlice)         | Equinix Metal |
+
+## Adoption
+
+We do not have an official ADOPTERS file. AWS and Equinix Metal are the only confirmed adopters that we are aware of. AWS is using Tinkerbell in its [EKS Anywhere](https://anywhere.eks.amazonaws.com/docs/getting-started/baremetal/overview/) product and is using parts of the Tinkerbell stack in its provisioning engine for its [Bare metal cloud](https://deploy.equinix.com/) offering. Our CNCF [slack channel](https://cloud-native.slack.com/archives/C01SRB41GMT) has almost 450 members from many different companies have used Tinkerbell in one form or another.
+
+There is one known open source project that has integrated Tinkerbell components into its product; [Seeder](https://github.com/harvester/seeder), which is a part of the Harvester project from Rancher.
+
+## Project goals
+
+As this is the first review for Tinkerbell, we don't have a previous set of goals to review. The projects current goals are to improve the user experience around trying out Tinkerbell, update the docs to catch up to the current state of the project, and to roll out the v1alpha2 API across all services. Details on these goals and the work in progress can be found in our [roadmap repository](https://github.com/orgs/tinkerbell/projects/16/views/1).
+
+## CNCF Membership
+
+Help with  
```

Comment:
This isn't necessarily a CNCF ask, but while thinking about this and how we can help ourselves through expanded awareness, I think we may want to migrate cluster-api-proivder-tinkerbell to kubernetes-sigs. 

I'm curious what other projects have asked for here.

Reply:
I'm not enthusiastic about contributing this to SIGs in its current state. There are a number of improvements I'd like to see post that should improve the UX and I think it would be easier to navigate those with the project under the Tinkerbell org.